plutus-core/plutus-core/src/PlutusCore/Constant/Meaning.hs

```diff
@@ -306,8 +343,8 @@ type SpecializeFromTo i j term a = HandleHole i j term (TypeHole a)
 -- 2. an uninstantiated costing function
 makeBuiltinMeaning
     :: forall a term cost binds args res j.
-       ( args ~ GetArgs a, a ~ FoldArgs args res, binds ~ Merge (ListToBinds args) (ToBinds res)
-       , KnownPolytype binds term args res a, SpecializeFromTo 0 j term a
+       ( binds ~ ToBinds a, args ~ GetArgs a, a ~ FoldArgs args res
```

Comment:
I realized I can now call `ToBinds` directly over `a`, no need to deconstruct `a` to get its bound variables.